### ISSUE TITLE
conda-forge version is outdated and does not install on Python 3.11.0 environments

### ISSUE THREAD

**Issue by accelyte** [pyapp-kit/psygnal#163]
### Description

psygnal fails to install via conda (https://anaconda.org/conda-forge/psygnal):
- the version on conda is outdated (should be 0.7.0, not 0.6.1)
- even with 0.6.1, conda fails to install psygnal due to my Python 3.11.0 environment (which again should not be the case)

### What I Did

```
>> mamba install psygnal==0.6.1

Looking for: ['psygnal==0.6.1']

conda-forge/win-64                                          Using cache
conda-forge/noarch                                          Using cache

Pinned packages:
  - python 3.11.*


Encountered problems while solving:
  - package psygnal-0.6.1-py39h1f6ef14_0 requires python >=3.9,<3.10.0a0, but none of the providers can be ...
```

**Comment by tlambert03** [comment]
thanks, will fix

**Comment by tlambert03** [comment]
Merged 0.7.0 feedstock.  Should publish within an hour or so
